intake/catalog/local.py

```diff
@@ -326,18 +261,25 @@ def _parse_plugins(self, data):
                 self.error("keys 'module' and 'dir' both exist (select only "
                            "one)", plugin_source)
             elif 'module' in plugin_source:
-                obj = self._parse_plugin(plugin_source, 'module')
-                if obj:
-                    sources.append(obj)
+                for k, v in load_plugins_from_module(
```

Comment:
For this block and the one below, it feels like there's enough here to warrant a separate function (`register_module` and `register_dir`).

Reply:
done